php/commands/cli.php

```diff
@@ -64,7 +64,12 @@ public function version() {
 	 * ## OPTIONS
 	 *
 	 * [--format=<format>]
-	 * : Accepted values: json
+	 * : Render output in a particular format.
+	 * ---
+	 * default: json
```

Comment:
The default format isn't `json` in this case. We need some name for the default format, and we should make sure there's test coverage for the default format (as the build didn't fail when it should've).